shell/webpack.config.js

```diff
@@ -190,7 +190,6 @@ module.exports = () => {
           ],
           use: [
             'thread-loader',
-            'babel-loader',
```

Comment:
could remove it from dependencies too.

Reply:
no hurry, I'm planning to replace ts-loader by babel-loader.  Because babel-loader can generate less code and easier control env-preset